lib/lbt/bundle/Builder.js

```diff
@@ -559,18 +600,16 @@ function rewriteDefine(targetBundleFormat, code, moduleName, avoidLazyParsing) {
 			}
 
 			// console.log("rewriting %s", module, defineCall);
-
 			return ast;
 		}
 
 
 		if ( isMethodCall(ast.body[0].expression, CALL_SAP_UI_DEFINE) ) {
 			const defineCall = ast.body[0].expression;
-
 			// rewrite sap.ui.define to sap.ui.predefine
 			if ( defineCall.callee.type === Syntax.MemberExpression
 					&& defineCall.callee.property.type === Syntax.Identifier
-					&& defineCall.callee.property.name === "define" ) {
+				&& defineCall.callee.property.name === "define") {
```

Comment:
move back to original position , please

Reply:
if you had prettier we wouldn't have this conversation ! (just teasing, yes I'll move it back)

Reply:
So 'prettier' basically is a tool that keeps people from talking to each other, did I get that right?